language/move-analyzer/editors/code/README.md

```diff
@@ -3,22 +3,75 @@
 Provides language support for the Move programming language.
 
 Currently, this means a basic grammar and language configuration for Move (`.move`) that enables
-syntax highlighting, commenting/uncommenting, and other basic language features in Move files.
+syntax highlighting, commenting/uncommenting, simple context-unaware completion suggestions while
+typing, and other basic language features in Move files.
 
-For information about Move, Diem, these projects' licenses, their contributor guides,
-and more, visit [the Diem repository](https://github.com/diem/diem).
+For information about Move, Diem, these projects' licenses, their contributor guides, links to our
+Discord server, and more, visit [the Diem repository](https://github.com/diem/diem).
 
-## How to Use
+## How to Install
+
+The move-analyzer Visual Studio Code extension works via two components: the extension itself, and
+the `move-analyzer` language server.
+
+### 1. Installing the `move-analyzer` language server<span id="Step1">
+
+The `move-analyzer` language server is a Rust program that is part of
+[the Diem repository](https://github.com/diem/diem). It may be installed in one of two ways:
+
+1. You may clone [the Diem repository](https://github.com/diem/diem) yourself and build
+   `move-analyzer` from its source code. This is recommended for Diem hackathon participants, and
+   Diem & Move core developers.
+   1. Follow the installation instructions in
+      [the Move tutorial](https://github.com/diem/diem/tree/main/language/documentation/tutorial#step-0-installation)
+      to install the project's prerequisite dependencies, which include `move-analyzer`.
```

Comment:
nit: "follow x to install the project's prerequisite dependencies" links to step 0, can we have the copy also say follow step 0? "project prequisite's" left me wondering which steps to follow. 

Furthermore, this link (https://github.com/diem/diem/tree/main/language/documentation/tutorial#step-0-installation) doesn't tell you how to install the `move-analyzer`, it does tell you how to use it tho:

> There is official Move support for VSCode, you can install this extension by opening VSCode and searching for the "move-analyzer" package and installing it. More detailed instructions can be found here.

link: https://github.com/diem/diem/tree/main/language/move-analyzer/editors/code

<img width="1160" alt="CleanShot 2021-12-03 at 15 15 14@2x" src="https://user-images.githubusercontent.com/635121/144684823-e7d2fa16-4d9d-4503-9d44-710d31638e4a.png">

Reply:
The following worked for me:

```bash
cargo install --path language/move-analyzer
```

Then I did the following to get the path for the next step for VS Code:

```bash
❯ which move-analyzer
/Users/droc/.cargo/bin/move-analyzer
```

Reply:
> nit: "follow x to install the project's prerequisite dependencies" links to step 0, can we have the copy also say follow step 0? "project prequisite's" left me wondering which steps to follow.

Will do!

```diff
-   1. Follow the installation instructions in [the Move tutorial](https://github.com/diem/diem/tree/main/language/documentation/tutorial#step-0-installation)
-      to install the project's prerequisite dependencies, which include `move-analyzer`.
+   1. Follow the instructions in the Move tutorial's [Step 0: Installation](https://github.com/diem/diem/tree/main/language/documentation/tutorial#step-0-installation).
```

> Furthermore, this link (https://github.com/diem/diem/tree/main/language/documentation/tutorial#step-0-installation) doesn't tell you how to install the move-analyzer

Along with the change below, I consider `cargo build -p df-cli -p move-analyzer` to be "installing." But I think I agree with you that the wording is vague. I think the new wording you helped me reach, written above, makes things clearer.

> The following worked for me: `cargo install --path language/move-analyzer`

I think this is better, but [the Move tutorial](https://github.com/diem/diem/tree/main/language/documentation/tutorial#step-0-installation) on the `main` branch seems to advocate for `cargo build -p df-cli`, then setting up an alias. I think your approach to use `cargo install` is simpler and better, but this feels like a broader discussion and change to the Move tutorial. Would you mind if I kept this for now and sent a separate pull request to change the Move tutorial to use `cargo install` instead? At that time I'll also switch to `cargo install --path language/move-analyzer`.

> Then I did the following to get the path for the next step for VS Code: `which move-analyzer`

JFYI, this step actually isn't necessary -- the default path for the language server is `move-analyzer`, and if that's in your `PATH` then it'll be used. So, no need to specify the absolute path to it as you did here! Still, thanks for trying all these steps out yourself!